x-pack/plugins/fleet/public/components/package_policy_delete_provider.test.tsx

```diff
@@ -7,7 +7,7 @@
 
 import React from 'react';
 
-import { act, fireEvent } from '@testing-library/react';
```

Comment:
removing instances of act wrapped around `fireEvent` shaved some seconds off the test running time here; 
### with act
<img width="1291" alt="Screenshot 2024-11-22 at 18 35 19" src="https://github.com/user-attachments/assets/b428667c-9847-49a4-acbb-538752b10a83">
## with waitFor
<img width="1286" alt="Screenshot 2024-11-22 at 18 39 20" src="https://github.com/user-attachments/assets/47426270-7b0f-4c07-967c-ca8a9828aa55">